blockbot/blockbot.php

```diff
@@ -58,10 +62,10 @@ function blockbot_init_1(App $a) {
 
 	foreach ($agents as $agent) {
 		if (stristr($_SERVER['HTTP_USER_AGENT'], $agent)) {
-			System::httpExit(403, 'Bots are not allowed');
+			System::httpExit(403, ['title' => 'Bots are not allowed']);
 		}
 	}
 
 	logger::info('Blocked bot', $logdata);
-	System::httpExit(403, 'Bots are not allowed');
+	System::httpExit(403, ['title' => 'Bots are not allowed']);
```

Comment:
The changed `System::httpExit` method in `develop` doesn't take an array as second parameter anymore. The previous version is actually right.